api/types/github.go

```diff
@@ -279,7 +232,7 @@ func (c *GithubConnectorV3) MapClaims(claims GithubClaims) ([]string, []string,
 			}
 		}
 	}
-	return utils.Deduplicate(roles), utils.Deduplicate(kubeGroups), utils.Deduplicate(kubeUsers)
+	return utils.Deduplicate(roles), nil, nil
```

Comment:
Looks like this is the only return in this function. Can we remove those arguments instead of always returning `nil`?

Reply:
Sure thing. I had a feeling someone would not like this :-)